server/data/data-jpa/src/test/java/org/apache/james/rrt/jpa/JPARecipientRewriteTableTest.java

```diff
@@ -57,18 +58,18 @@ protected AbstractRecipientRewriteTable getRecipientRewriteTable() throws Except
     }
 
     @Override
-    protected void addMapping(String user, Domain domain, String mapping, int type) throws RecipientRewriteTableException {
+    protected void addMapping(String user, Domain domain, String mapping, Type type) throws RecipientRewriteTableException {

```

Comment:
`addMapping` and `removeMapping` seems to be repeated throughout the code base.

Maybe it would make sens to factorize it somehow?